tools/count_vale_errors.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+vale -v 
+echo "# Breakdown of vale errors per module:"
+
+for module in modules/*
+    do
+    echo -n "$module: "
```

Comment:
it doesn't pass shellcheck, so probably use bash instead of sh

Reply:
Oh that's strange, shell check was passing locally. I'll fix it after some PTO.

Reply:
Indeed. Last minute change introduced the error. Nicely spotted. Fixed.